patterns/direct_replier.py

```diff
@@ -0,0 +1,122 @@
+## Goal: A simpe request pulblisher to publish a request with reply_to set to any topic.
+# Unlike regular publisher, this will be built on RequestReplyMessagePublisher which exposes the ability to set 'Reply To' field
+import os
+import platform
+import time
+import calendar;
+
+# Import Solace Python  API modules from the solace package
+from solace.messaging.messaging_service import MessagingService, ReconnectionListener, ReconnectionAttemptListener, \
+            ServiceInterruptionListener, RetryStrategy, ServiceEvent
+from solace.messaging.resources.topic import Topic
+from solace.messaging.resources.topic_subscription import TopicSubscription
+from solace.messaging.receiver.request_reply_message_receiver import RequestMessageHandler, InboundMessage, Replier
+
+if platform.uname().system == 'Windows': os.environ["PYTHONUNBUFFERED"] = "1" # Disable stdout buffer 
+
+# Goal is to demonstrate request-reply pattern using Direct messages. A request message carrying {name} is published,
+# and a response with a "Greetings {name}" is expected as response.
+#   direct_requestor: publishes('solace/samples/python/direct/hello/{name}, and waits for response (either synchronously or asynchronously)
+#   direct_replier: subscribes('solace/samples/python/direct/hello/>'), and replies with a greetings message in the body.
+
+TOPIC_PREFIX = "solace/samples/python"
+
+# Handle received messages
+class RequestMessageHandlerImpl(RequestMessageHandler):
+    def on_message(self, request: InboundMessage, replier: Replier):
+        name = request.get_payload_as_string().split("My name is ")[1]
```

Comment:
This also assumes that the request payload is string

Reply:
Done!